pom.xml

```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>3.1.5</version>
+		<relativePath/> <!-- lookup parent from repository -->
+	</parent>
```

Comment:
There is a Companies House parent-pom which sets up some sonar specific config and use of the artifactory (I think concourse does do this anyway).

If keeping the spring-boot-starter-parent then maybe worth setting those sonar properties up in the local pom?

Reply:
I tried switching this out for the CH parent but the application wouldn't start but no error given.
It might have been unrelated but it is currently ok without that change.
Maybe postpone any sonar changes till the pipeline work?

Reply:
I think it's probably at least worth setting the sonar config up ahead of time? I can approve without this and include in the tech debt ticket?

Reply:
I can do it as a separate PR under the current ticket